core/templates/dev/head/css/oppia.css

```diff
@@ -2873,3 +2873,514 @@ md-card.preview-conversation-skin-supplemental-card {
   color: red;
   margin-bottom: 10px;
 }
+
+.conversation-skin-progress-dots {
```

Comment:
I thought about moving styles to separate css but i haven't seen any mechanism for loading many CSS files. I think it will be easy to move this styles to the separated part later.